addons/l10n_se/data/account_fiscal_position_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- ************************************************************* -->
+<!-- Fiscal Position Templates -->
+<!-- ************************************************************* -->
+<odoo>
+    <data noupdate="1">
+        <record id="fp_sweden" model="account.fiscal.position.template">
```

Comment:
@daniel-stenlov "Sverige" fiscal position does not use any ware?
any special reason to add this?

Reply:
The only reason is to get it more transparent for the end user. However system vice it's not really used since basic setup is SWEDEN.

Reply:
It's also a way to see if it's set or not set. So you can for example make a list of not set customers and by this editing them only. You can see it as a Fuzzy Logic setting, TRUE, FALSE, NOT SET.

Reply:
@daniel-stenlov good thing, but this is not really used so we removed it.